docs/source/references/modules.rst

```diff
@@ -410,6 +410,12 @@ UniversalImageQualityIndex
 .. autoclass:: torchmetrics.UniversalImageQualityIndex
     :noindex:
 
+SpectralAngleMapper
```

Comment:
again, please move such that the list of metrics (within the image subsection) is in alphabetical order